state/entrySyncing.go

```diff
@@ -317,7 +320,7 @@ func (s *State) GoSyncEntries() {
 		}
 		lastfirstmissing = firstMissing
 		if firstMissing < 0 {
-			s.EntryDBHeightComplete = s.GetHighestSavedBlk()
+			s.EntryDBHeightComplete = highestSavedBlk
 			s.LogPrintf("EntrySync", "firstMissing EntryDBHeightComplete = %d", s.EntryDBHeightComplete)
```

Comment:
Is the intention to log this line on every iteration? (In the old scenario like every 10ms)
If not I think it should be
if firstMissing < 0 {
			if s.EntryBlockDBHeightComplete < highestSavedBlk {
				s.EntryDBHeightComplete = highestSavedBlk
				s.LogPrintf("EntrySync", "firstMissing EntryDBHeightComplete = %d", s.EntryDBHeightComplete)
			}
			time.Sleep(10 * time.Millisecond)
		}